templates/release-post.adoc

```diff
@@ -217,6 +217,22 @@ For more details, check the LINK[LINK_DESCRIPTION].
 // // // // // // // //
 
 
+[#bugs]
+== Significant bugs fixed in this release
+
+
+We’ve spent some time fixing bugs. The following sections describe just some of the issues we resolved in this release. If you’re interested, here’s the full list of fixed bugs in 20.0.0.RELEASE_VERSION.
```

Comment:
`some of the issues we resolved in this release` -> `some of the issues resolved in this release`

or

`some of the issues we resolved in this release` -> `some of the issues we've resolved in this release`

Reply:
Changed to the first one.